index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="header">
+    <div class="up"><img src="icons/Dark.png"></div>
+    <div class="logo">My Logo</div>
+    <div class="union"><img src="icons/Union.png"></div>
+  </div> 
+  <div class="headline">
+    <div class="text">Hey There. <br> I'm Alex.</div>
+    <div class="text2">I'm a Software Developer</div>
+    <div class="text3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</div>
+    <div class="icon1"><img src="icons/icon1.png"></div>
+    <div class="icon2"><img src="icons/icon2.png"></div>
+    <div class="icon3"><img src="icons/icon3.png"></div>
+    <div class="icon4"><img src="icons/icon4.png"></div>
+    <div class="icon5"><img src="icons/icon5.png"></div>
```

Comment:
Kindly update your list of social links to be wrapped in a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this:

`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`